spec/helpers/court_dates_helper_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe CourtDatesHelper do
+  describe "#when_do_we_have_court_dates" do
+    describe "when casa case has only dates in the past" do
+      let(:casa_case) { create(:casa_case, :with_past_court_date) }
+
+      it "returns 'past'" do
+        expect(helper.when_do_we_have_court_dates(casa_case)).to eq("past")
+      end
```

Comment:
**nitpick:** I tend to think these types of tests a bit too repetitive, the case description adds very little to the actual code being executed. In these cases I prefer something terser:

```rb
it { expect(helper.when_do_we_have_court_dates(casa_case)).to eq("past") }
```

That said, up to taste :)